app/javascript/hooks/mutations/admin/server_recordings/useRecordingsReSync.jsx

```diff
@@ -0,0 +1,24 @@
+import { t } from 'i18next';
+import toast from 'react-hot-toast';
+import { useMutation, useQueryClient } from 'react-query';
+
+import axios from '../../../../helpers/Axios';
+
+export default function useRecordingsReSync(friendlyId) {
+  const queryClient = useQueryClient();
+
+  return useMutation(
+    () => axios.get(`/admin/server_rooms/${friendlyId}/resync.json`),
```

Comment:
This is a great example of an API that mutates our server(s) state and vulnerable to CSRF attacks even with our mitigation,

Reply:
Agreed